front/lib/resources/group_resource.ts

```diff
@@ -680,23 +680,35 @@ export class GroupResource extends BaseResource<GroupModel> {
 
   // Permissions
 
-  acl(): ACLType {
-    return {
-      aclEntries: [
-        {
-          groupId: this.id,
-          permissions: ["read"],
-        },
-      ],
-    };
+  /**
+   * Returns the requested permissions for this resource.
+   *
+   * Configures two types of access:
+   * 1. Group-based: The group's members get read access
+   * 2. Role-based: Workspace admins get read and write access
+   *
+   * @returns Array of ResourcePermission objects defining the default access configuration
+   */
+  requestedPermissions(): ResourcePermission[] {
```

Comment:
Had to keep it it's used to check proper permissioning when adding/removing members.

Reply:
`requested` is confusing to me (i expect to return something like ["read"] or ["read", "write"]).

I don't have a better idea top of my head but I'm sure we can do better.